### DISCUSSION TITLE
Adding Options

### DISCUSSION THREAD

**Discussion by audiokid** [katspaugh/wavesurfer.js#3409] [Q&A]
Hi waversurfers, I'm new to the community and need some help adding the Options. I've installed waversurefer.js to my Drupal 9 website but can only get the basic wave to show. I see all the plugins but they don't work for me. 
Is it possible to add more options like the zoom, regions etc for my members? Here is an example of what I have working. https://recording.org/forum/mixing-song-critique/my-mix-any-good Any help would be greatly appreciated. 

Thank you.

**Comment by katspaugh** [comment]
You should include the plugins' scripts along with the wavesurfer script like here: https://github.com/katspaugh/wavesurfer.js/blob/main/cypress/e2e/umd.html#L8-L10

When inserted like this, the plugins are available inside the WaveSurfer namespace. E.g. `WaveSurfer.Regions`.

**Comment by audiokid** [comment]
Thanks for your help! 
Is this included with the wavesurfer.js package ( https://unpkg.com/browse/wavesurfer.js@7.5.1/ ) or another directory?
Do I simply need to add the lines or am I missing a directory that needs to be added somewhere eg: libraries/wavesurfer/dist/ 

?

**Reply by katspaugh** [reply]
Yes, it’s in that package.

**Reply by audiokid** [reply]
I've tried adding this in the main folder ( libraries/wavesurfer/   and also in the waveburner/dist/ directories but drupal doesn't see it.  Are you familiar with drupal? 

Thanks again for any help.

**Reply by audiokid** [reply]
PS, this is the drupal module that adds the frontend for wavesurfer to display in drupal https://www.drupal.org/project/audiofield 
I'm a novice coder that relies on others who build modules so I'm at a loss to figure out what could be a simple fix. 

I'm wondering now if the drupal audiofield module is missing the ability to find the extra wavesurfer options.
